src/en/installconfig-os-support.md

```diff
@@ -0,0 +1,70 @@
+Title: Operating System Support
+TODO: This document needs to be replaced or substantially updated
+      maas-image-builder section removed
+      Adding documentation for building custom images is maas-docs issue #17
+
+# Operating Systems Support
+
+Besides Ubuntu, MAAS allows users to deploy different operating systems, such
+as CentOS. Ubuntu Advantage customers can additionally deploy Red Hat
+Enterprise Linux (RHEL), OpenSUSE, SLES, Window Server and Windows HyperV.
```

Comment:
Comments on this paragraph.

- "allows users to deploy different operating systems, such as CentOS". We should just state explicitly what they are.
- "Ubuntu Advantage customers". This will confuse the casual reader. I suggest "Canonical Support customers".
- "OpenSUSE" should be "openSUSE"
- "Windows Server" is a brand name. The operating system is like "Windows Server 2012". What does MAAS support?
- "Hyper-V" (sic) is a component of some Server edition, so "Hyper-V Server 2008". What does MAAS support?